index.bs

```diff
@@ -1419,18 +1419,18 @@ disabled, the return value is always empty.
 
   1. Let |event map| be a new map.
 
-  1. For each |key| → |value| of the [=browsing context event map=] for
-     |session|:
+    1. For each |key| → |value| of the [=browsing context event map=] for
```

Comment:
I don't understand why this is indented. The step above doesn't introduce control flow (either a conditional or a loop), which is usually why we have nested steps.

Reply:
We do the same for `event names` right after. There is also a for loop, which caused an indentation. So why is that other indentation correct, while this is not?

Reply:
I think the other one is wrong and this is right (my guess is the other one was originally two seperate steps, one for the loop and one for the inner steps and when it got squashed into a single loop I accidentially carried over the indentation from the inner steps).